### ISSUE TITLE
Hide-Toggle on IE11

### ISSUE THREAD

**Issue by coliff** [sandoche/Darkmode.js#11]
I think it'd be good to make this run at all on IE11 and stop the widget icon from displaying. Could maybe do this in JavaScript or use CSS targeting.

I use this to hide in IE10/IE11
```css
@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
.darkmode-toggle {display: none !important}
}
```

**Comment by sandoche** [comment]
Good idea I'll add it in the next release!

**Comment by sandoche** [comment]
I will also add this for Edge
```
      @supports (-ms-ime-align:auto), (-ms-accelerator:true) {
        .darkmode-toggle {display: none !important}
      }
```

**Comment by sandoche** [comment]
It's now implemented in the version 1.4.0
